ui/StatusQ/src/StatusQ/Components/StatusMessage.qml

```diff
@@ -253,7 +252,9 @@ Control {
                     }
                     Loader {
                         Layout.fillWidth: true
-                        active: !root.editMode && !!root.messageDetails.messageText && !root.hideMessage
+                        active: (!root.editMode && !!root.messageDetails.messageText && !root.hideMessage
+                             && ((root.messageDetails.contentType === StatusMessage.ContentType.Text)
+                             || (root.messageDetails.contentType === StatusMessage.ContentType.Emoji)))
```

Comment:
This will break messages imported from Discord, which have to render the text component, but are not of contenttype `Text` (but `DiscordMessage`).

@alexandraB99 is it possible to find a fix, that doesn't required a constraint on this content type?

Reply:
@0x-r4bbit added the discord message case, you might wanna have a second look!